cloud_governance/common/clouds/aws/sts/sts_oprations.py

```diff
@@ -0,0 +1,14 @@
+import boto3
+
+
+class STSOperations:
+
```

Comment:
Please add documentation for this class

Reply:
yes, It will also gives the caller identity

Reply:
So please add it to documentation